src/Exceptions/CouldNotSendNotification.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace NotificationChannels\Workplace\Exceptions;
+
+use GuzzleHttp\Exception\ClientException;
+
+class CouldNotSendNotification extends \Exception
+{
+    /**
+     * Thrown when there's a bad request and an error is responded.
+     *
+     * @param ClientException $exception
+     *
+     * @return static
+     */
+    public static function workplaceRespondedWithAnError(ClientException $exception)
+    {
+        $statusCode = $exception->getResponse()->getStatusCode();
+        $description = 'no description given';
+        if ($result = json_decode($exception->getResponse()->getBody())) {
+            $description = $result->description ?: $description;
```

Comment:
```suggestion
            $description = $result->description ?? 'No description given';
```

Reply:
I've kept the `$description = 'no description given';` outside the if with the default value but used the null coalescing operator